### PR TITLE
fix: lowercase attributes

### LINKED ISSUE #483: [scoped-custom-elements-registry] Setting attributes doesnt work as expected when polyfill enabled
### Description

Using `lit-element@2.4.0` and `@open-wc/scoped-elements@1.3.4` we're noticing some unexpected behavior when setting boolean attributes. The following code:

```js
return html`<scope-me ?fooBar=${true}></scope-me>`;
```

**Behavior _without_ loading the polyfill:**
Sets a `foobar` attribute on `<scope-me>` and it also sets the `fooBar` property to `true`

**Behavior _with_ loading the polyfill:**
Sets a `foobar` attribute on `<scope-me>`, but the `fooBar` property is `undefined`

### Example

```js
/**
 * Specific version:
 *
 * @webcomponents/scoped-custom-element-registry@0.0.3
 * @open-wc/scoped-elements@1.3.4
 * lit-element@2.4.0
 */

// Disabling this import will make `ScopeMe` render ✅
import '@webcomponents/scoped-custom-element-registry';

import { LitElement, html } from 'lit-element';
import { ScopedElementsMixin } from '@open-wc/scoped-elements';

class ScopeMe extends LitElement {
  static get properties() {
    return {
      fooBar: { type: Boolean },
    };
  }

  render() {
    return html`hello ${this.fooBar ? '✅' : '❌'}`;
  }
}

class MyEl extends ScopedElementsMixin(LitElement) {
  static get scopedElements() {
    return {
      'scope-me': ScopeMe,
    };
  }

  render() {
    return html`<scope-me ?fooBar=${true}></scope-me>`;
  }
}

customElements.define('my-el', MyEl);

```

[Stackblitz repro](https://stackblitz.com/edit/js-rjfm8m?file=index.js)

### Version
0.0.3

### Browsers affected

- [x] Chrome
- [ ] Firefox
- [ ] Edge
- [ ] Safari
- [ ] IE 11

### PR DESCRIPTION
**Pull request by thepassle**
Fixes #483

### CODE REVIEW

**Review by aomarks — APPROVED** [review]

File: packages/scoped-custom-element-registry/CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed form-associated custom element definitions (`static formAssociated = true`) when used with the polyfill.
 - Fixed patched callback names in form-associated custom element support.
 
+## [0.0.4] - 2022-26-01
+
+- Lowercasing of attributes. Fixes #483
```
Comment (aomarks) [inline packages/scoped-custom-element-registry/CHANGELOG.md:17]:
```suggestion
- Fixed handling of mixed-case attributes. Fixes #483
```

Comment (aomarks) [reply packages/scoped-custom-element-registry/CHANGELOG.md:17]: Note we create the release sections when we release, so for now it should stay under Unreleased.

File: packages/scoped-custom-element-registry/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@webcomponents/scoped-custom-element-registry",
-  "version": "0.0.3",
+  "version": "0.0.4",
```
Comment (aomarks) [inline packages/scoped-custom-element-registry/package.json:3]:
```suggestion
  "version": "0.0.3",
```

We bump versions when we release

**Review by kevinpschaaf — APPROVED** [review]

**Comment by thepassle** [comment]
@kevinpschaaf @aomarks Could you cut a release for this version? I see it hasnt been released yet: https://www.npmjs.com/package/@webcomponents/scoped-custom-element-registry

**Comment by michaelwarren1106** [comment]
I'm also having boolean attribute problems with the ScopedElementsMixin. Is there any chance of a 0.0.5 release soon?

**Pull request merged**